sdk/webpubsub/azure-webpubsub-upstream/tests/test_exception.py

```diff
@@ -0,0 +1,8 @@
+# coding=utf-8
```

Comment:
Delete sdk/webpubsub/azure-webpubsub-upstream/tests/__init__.py, a tests folder is not a package